src/less/day-slot.less

```diff
@@ -44,6 +44,30 @@
   &:nth-child(even){
     border-top-color: lighten(@cell-border, 10%);
   }
+
+  &.three {
```

Comment:
I think that this can be handled via flex box styles for any slice amount, which would be better then needing hard coded values